UM-USElab/gradecraft-development
# Refactor grading status

### Status
**READY**

### Description
This work is a refactor of the grading status page, so that it improves user experience by loading each of the four sections asynchronously (Ungraded submissions, In Progress Grades, Ready for Release Grades, and Resubmissions)

### Migrations
NO

### Steps to Test or Reproduce
Ensure that each of the subsections load as intended and that all table features work as expected (sorting, search, valid links, ability to release grades, etc). The number of loaded records per subsection should also match the count.

### Impacted Areas in Application
Grading status page

======================
Relates to #3880
